google-osconfig-agent.service

```diff
@@ -6,7 +6,7 @@ Wants=local-fs.target network-online.target network.target
 [Service]
 ExecStart=/usr/bin/google_osconfig_agent
 Restart=always
-RestartSec=120
+RestartSec=1
```

Comment:
what's the background for this? are you sure you always want instant restarts? what's systemd behavior for infinite loops?

Reply:
Fixed with StartLimitInterval and StartLimitBurst